lib/boom_notifier/application.ex

```diff
@@ -6,7 +6,11 @@ defmodule BoomNotifier.Application do
   def start(_type, _args) do
     import Supervisor.Spec
 
-    children = [supervisor(BoomNotifier.ErrorStorage, [])]
+    children = [
+      supervisor(BoomNotifier.ErrorStorage, []),
+      {BoomNotifier.NotifierSenderServer, nil}
```

Comment:
Super minor thing, but wondering if it would be a tiny bit more clear, when reading this code, to understand that the init argument "doesn't matter" if we just pass `BoomNotifier.NotifierSenderServer` instead of the tuple, and let the default argument be passed implicitly?

Reply:
Nice, I didn't know you could do that 💪